index.js

```diff
@@ -6,83 +6,173 @@
  * Licensed under the MIT license.
  */
 
-module.exports = {
-	sets: {
-		ascii: " A1B\'K2L@CIF/MSP\"E3H9O6R^DJG>NTQ,*5<-U8V.%[$+X!&;:4\\0Z7(_?W]#Y)=",
-		dot6:  "⠀⠁⠂⠃⠄⠅⠆⠇⠈⠉⠊⠋⠌⠍⠎⠏⠐⠑⠒⠓⠔⠕⠖⠗⠘⠙⠚⠛⠜⠝⠞⠟⠠⠡⠢⠣⠤⠥⠦⠧⠨⠩⠪⠫⠬⠭⠮⠯⠰⠱⠲⠳⠴⠵⠶⠷⠸⠹⠺⠻⠼⠽⠾⠿"
-	},
-
-	// converts character to Braille symbol
-	convert: function (character) {
-		var sign = undefined;
+var BRAILLE = {
+    " ": " ",
+    "_": "⠸",
+    "-": "⠤",
+    ",": "⠠",
+    ";": "⠰",
+    ":": "⠱",
+    "!": "⠮",
+    "?": "⠹",
+    ".": "⠨",
+    "(": "⠷",
+    "[": "⠪",
+    "@": "⠈",
+    "*": "⠡",
+    "/": "⠌",
+    "\'": "⠄",
+    "\"": "⠐",
+    "\\": "⠳",
+    "&": "⠯",
+    "%": "⠩",
+    "^": "⠘",
+    "+": "⠬",
+    "<": "⠣",
+    ">": "⠜",
+    "$": "⠫",
+    "0": "⠴",
+    "1": "⠂",
+    "2": "⠆",
+    "3": "⠒",
+    "4": "⠲",
+    "5": "⠢",
+    "6": "⠖",
+    "7": "⠶",
+    "8": "⠦",
+    "9": "⠔",
+    "A": "⠁",
+    "B": "⠃",
+    "C": "⠉",
+    "D": "⠙",
+    "E": "⠑",
+    "F": "⠋",
+    "G": "⠛",
+    "H": "⠓",
+    "I": "⠊",
+    "J": "⠚",
+    "K": "⠅",
+    "L": "⠇",
+    "M": "⠍",
+    "N": "⠝",
+    "O": "⠕",
+    "P": "⠏",
+    "Q": "⠟",
+    "R": "⠗",
+    "S": "⠎",
+    "T": "⠞",
+    "U": "⠥",
+    "V": "⠧",
+    "W": "⠺",
+    "X": "⠭",
+    "Z": "⠵",
+    "]": "⠻",
+    "#": "⠼",
+    "Y": "⠽",
+    ")": "⠾",
+    "=": "⠿"
+  },
 
-		// searches sets
-		for (var i = this.sets.ascii.length - 1; i >= 0; i--) {
-			if(this.sets.ascii[i] == character){
-				sign = this.sets.dot6[i];
-			}
-		}
-		
-		// if not found will return undefined
-		return sign;
-	},
+  ASCII = {
+    " ": " ",
+    "⠸": "_",
+    "⠤": "-",
+    "⠠": ",",
+    "⠰": ";",
+    "⠱": ":",
+    "⠮": "!",
+    "⠹": "?",
+    "⠨": ".",
+    "⠷": "(",
+    "⠪": "[",
+    "⠈": "@",
+    "⠡": "*",
+    "⠌": "/",
+    "⠄": "\'",
+    "⠐": "\"",
+    "⠳": "\\",
+    "⠯": "&",
+    "⠩": "%",
+    "⠘": "^",
+    "⠬": "+",
+    "⠣": "<",
+    "⠜": ">",
+    "⠫": "$",
+    "⠴": "0",
+    "⠂": "1",
+    "⠆": "2",
+    "⠒": "3",
+    "⠲": "4",
+    "⠢": "5",
+    "⠖": "6",
+    "⠶": "7",
+    "⠦": "8",
+    "⠔": "9",
+    "⠁": "A",
+    "⠃": "B",
+    "⠉": "C",
+    "⠙": "D",
+    "⠑": "E",
+    "⠋": "F",
+    "⠛": "G",
+    "⠓": "H",
+    "⠊": "I",
+    "⠚": "J",
+    "⠅": "K",
+    "⠇": "L",
+    "⠍": "M",
+    "⠝": "N",
+    "⠕": "O",
+    "⠏": "P",
+    "⠟": "Q",
+    "⠗": "R",
+    "⠎": "S",
+    "⠞": "T",
+    "⠥": "U",
+    "⠧": "V",
+    "⠺": "W",
+    "⠭": "X",
+    "⠵": "Z",
+    "⠻": "]",
+    "⠼": "#",
+    "⠽": "Y",
+    "⠾": ")",
+    "⠿": "="
+  };
 
-	// converts Braille symbol to letter
-	read: function (symbol) {
-		var letter = undefined;
-
-		// searches sets
-		for (var i = this.sets.ascii.length - 1; i >= 0; i--) {
-			if(this.sets.dot6[i] == symbol){
-				letter = this.sets.ascii[i];
-			}
-		}
-		
-		// if not found will return undefined
-		return letter;
-	},
+module.exports = {
+  convert: function (character) {
+    return !!BRAILLE[character] ? BRAILLE[character] : " ";
```

Comment:
Ok, we have to think first what to return if there is a character without equivalence, question mark, blank space or something else. What I was doing was to check if the property exists and that doesn't contain a falsy value (the double `!!` coerces the value to a boolean). If you want to test that a key exists, even if its value is falsy, then is ok to use `Object.prototype.hasOwnProperty.call(BRAILLE, i)`.  Hope it helped :)